tutorials/cloud-iot-rtdp/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Real time Data Processing With Google Cloud IoT
+description: Demonstrate how to build data processing backend for Google Cloud IoT.
+author: teppeiy
+tags: Cloud IoT Core, Cloud Dataflow, Cloud Functions, Java, Node.js
+date_published: 2017-11-29
+---
+# Real time Data Processing With Google Cloud IoT
+This repository contains sample code for Real time Data Processing With Google Cloud IoT solution. Sample MQTT client generates simulated data and submits them to Cloud IoT Core. Cloud Functions and Cloud Dataflow receive data through Cloud Pub/Sub.
+
+![Architecture](arch.png)
+
+## Prerequisites
+- Git
+- Google Cloud SDK
+- Java 1.8 or later
+- Apache Maven 3.3.x or later
+
+All commands in following steps are assumed to be executed under "community/tutorials/cloud-iot-rtdp" directory.
```

Comment:
This is the directory that we're in in the repo, yes? So we are assuming that they'll clone the repo and then run the tutorial in place?

Reply:
Yes, that's the assumption. Will add a comment to clone the repo first.